### PR TITLE
added my first commit to aviyel

### PR DESCRIPTION
**Pull request by spykelionel**
Added my first PR

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -53,7 +53,7 @@
 51. [Awosise Oluwaseun](#awosise-oluwaseun)
 52. [Issac Jacob](#issac-jacob)
 53. [Adeola Adetunji](#adeola-adetunji)
-
+54. [Ndi Lionel](#spykelionel)
```
Comment (siddharth2798) [inline CONTRIBUTORS.md:56]: Please hyperlink your name with the H2 heading you added above your detailed bio

**Review by siddharth2798 — CHANGES_REQUESTED** [review]
Please hyperlink your name with the H2 heading you added above your detailed bio
Refer other entries

File: CONTRIBUTORS.md (hunk 1 above)
Comment (spykelionel) [reply CONTRIBUTORS.md:56]: Added hyperlink to name

**Comment by spykelionel** [comment]
@siddharth2798  Please kindly look into this

**Review by RakeshPotnuru — APPROVED** [review]

**Review by pramit-marattha — APPROVED** [review]

**Review by pramit-marattha — APPROVED** [review]

**Comment by pramit-marattha** [comment]
Closing this one; please submit a new PR.

**Pull request closed without merging**